### DISCUSSION TITLE
Can sub-addressed e-mail addresses, added during creation, be modified?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [anonaddy/anonaddy#460] [Q&A] [answered]
Reading [`anonaddy.com/blog/add-recipients-to-aliases-at-creation-and-new-uuid-aliases`](https://anonaddy.com/blog/add-recipients-to-aliases-at-creation-and-new-uuid-aliases/), I wondered whether the user could change the keys subsequent to creation of the address.

**Comment by willbrowningme** [comment, marked as answer]
No, they are only added when the alias is first created. You will need to login and change the recipients after that.
